doc/Language/nativecall.pod6

```diff
@@ -185,6 +185,19 @@ When the signature of your native function needs a pointer to some native type
     # C prototype is void my_version(int *major, int *minor)
     sub my_version(int32 is rw, int32 is rw) is native('foo') { * }
     my_version(my int32 $major, my int32 $minor); # Pass a pointer to
+    
+When you just want to use/pass a pointer using Pointer class you have to 
+create the pointers when you declare them. Then you can do this without specifying 
+what type it points to. Here is the exmaple:
+    use NativeCall;
```

Comment:
Can you please separate that with a newline? It's probably not going to work. Explicit fencing is even better.